<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>570</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QToolBox" name="toolBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>831</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="toolBoxPage1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>831</width>
      <height>449</height>
     </rect>
    </property>
    <attribute name="icon">
     <iconset theme="call-stop"/>
    </attribute>
    <attribute name="label">
     <string>磁盘文件扫描</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>获取磁盘文件属性</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>711</width>
       <height>371</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>831</width>
      <height>449</height>
     </rect>
    </property>
    <attribute name="label">
     <string>SQL语句生成</string>
    </attribute>
    <widget class="QLineEdit" name="sqlpath">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>58</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>请输入文件存放路径：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>30</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>确认生成</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>751</width>
       <height>351</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>0</y>
       <width>301</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700; color:#ff0000;&quot;&gt;注意！完成磁盘文件扫描后再进行SQL语句生成&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="toolBoxPage3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>831</width>
      <height>449</height>
     </rect>
    </property>
    <attribute name="label">
     <string>目录树模拟操作</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>801</width>
       <height>471</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>801</width>
       <height>441</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>801</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>统计目录信息</string>
      </attribute>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>141</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>要统计目录的绝对路径：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>161</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>从文件中导入（文件地址）:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>10</y>
         <width>281</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>50</y>
         <width>281</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>10</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>统计</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>90</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>161</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>统计信息存放路径：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>90</y>
         <width>281</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>140</y>
         <width>711</width>
         <height>261</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>文件模拟操作</string>
      </attribute>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>40</y>
         <width>211</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>40</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>myfile.txt地址：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>80</y>
         <width>91</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>mystat文件路径：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>80</y>
         <width>211</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>40</y>
         <width>92</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>确认修改</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser_4">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>150</y>
         <width>701</width>
         <height>241</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>110</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>文件更改信息存放路径：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>110</y>
         <width>211</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="filechangediff">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>110</y>
         <width>91</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>检查修改</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>目录模拟操作</string>
      </attribute>
      <widget class="QLineEdit" name="lineEdit_7">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>30</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>30</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>mydir.txt地址：</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser_5">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>150</y>
         <width>631</width>
         <height>261</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_7">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>60</y>
         <width>92</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>确认删除</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>70</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>删除文件信息地址：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_8">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>70</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_8">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>110</y>
         <width>161</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>再次统计文件信息比较差异</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
